fml/mapping.h

```diff
@@ -125,6 +125,44 @@ class NonOwnedMapping final : public Mapping {
   FML_DISALLOW_COPY_AND_ASSIGN(NonOwnedMapping);
 };
 
+/// A Mapping like NonOwnedMapping, but uses Free as its release proc.
+class MallocMapping final : public Mapping {
+ public:
+  MallocMapping();
+
+  MallocMapping(const uint8_t* data, size_t size);
+
+  MallocMapping(fml::MallocMapping&& mapping);
+
+  ~MallocMapping() override;
+
+  /// Copies the data from `begin` to `end`.
+  /// It's templated since void* arithemetic isn't allowed and we want support
+  /// for `uint8_t` and `char`.
+  template <typename T>
+  static MallocMapping Copy(const T* begin, const T* end) {
+    size_t length = end - begin;
+    return Copy(begin, length);
+  }
+
+  static MallocMapping Copy(const void* begin, size_t length);
+
+  // |Mapping|
+  size_t GetSize() const override;
+
+  // |Mapping|
+  const uint8_t* GetMapping() const override;
+
+  /// Removes ownership of the data buffer.
+  const uint8_t* Release();
```

Comment:
I don't think this should be const because the caller won't be able to modify it (for freeing mostly) without a const cast. If this object is relinquishing ownerhip of the buffer, why should it have a say in how the caller uses the buffer that it now owns.

Reply:
`[[nodiscard]]` attribute here since ignoring this will be a leak.

Reply:
Added no discard.  Yea, you're right about the constness, remove it.